contracts/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
no need for this file, (almost) all the repo follows the same LICENSE specified in the root folder